app/routes/models/search.js

```diff
@@ -0,0 +1,26 @@
+function ViewModel (labelText, frn, error) {
```

Comment:
Nice job on making this as generic as poss by including custom `labelText` but could the name and value, i.e. `frn` be named more generic too?

Reply:
frn has been changed to value